discovery/triton/triton.go

```diff
@@ -219,6 +229,12 @@ func (d *Discovery) refresh() (tg *targetgroup.Group, err error) {
 		}
 		addr := fmt.Sprintf("%s.%s:%d", container.VMUUID, d.sdConfig.DNSSuffix, d.sdConfig.Port)
 		labels[model.AddressLabel] = model.LabelValue(addr)
+
+		if container.Groups != nil && len(container.Groups) > 0 {
+			name := strings.Join(container.Groups, ",")
+			labels[tritonLabelGroups+"_"+model.LabelName(","+name+",")] = model.LabelValue(name)
```

Comment:
The name should be fixed, look at how Consul tags are handled

Reply:
Okay, I understand now. Fixing this currently.